### DISCUSSION TITLE
upgraded and it dies after an hour or so now

### DISCUSSION THREAD

**Discussion by benjackson1111** [aqualinkd/AqualinkD#160] [General]
So, firstly, thank you for this. Like others, this brought my old pool tech into the 21st century and Im so grateful.

I have a $250 worth of high end cable running from my raspberry pi all the way out to my board, AND I upgraded the board to a proper one that didn't have to rely on PDA mode. Once I did all that, this always worked brilliantly for me.

The other day, I was h aving some other trouble with hard drives on my pi and in the process of pulling things in and out, the cable fell out from one of the pins. In the process of trying to troubleshoot, I tried to update the aqualinkd software and check all settings, check the USB serial logger etc, and found that was the problem. Fixing the cable brought everything back, and serial logger was showing things approprlately again. Wohoo - no worries.

However, ever since then? aqualinkd stops working. no errors. The web interface says connected, but it's no longer showing any updates as the programming  happens. So when the pool filter is on, it doesn't show, when the waterfall comes on at 2pm for 10 minutes, it doesn't show. If I restart the aqualinkd service, it comes back and everything works. But only for an hour or so (i'm guessing). I restarted it this morning at 12:57 just before my pool filter was to turn off, then by the time the waterfall came on at 2pm it wasn't allowing me to see it or turn it off manually etc.

What on earth could have happened? Any  hints? I'm pretty good at troubleshooting things so don't want to burden you with logs or my setup. I just was hoping maybe for a suggestion for me to look into as it's very very strange.

Thank you!

**Comment by benjackson1111** [comment]
As what usually happens, when I write out the issue like I did, it helps me answer my own question. Not sure if I've solved it but this is what I saw. I checked  the config file in /etc/ and it seemed to have defaulted back to default, probably when i did a reinstall/upgrade of the software. I noticed that the default was RS-8 whereas my panel is RS-6. So I copy and pasted my original config file. See how I go. I guess I didn't think to look here first because it WAS working when I first restarted teh service, it only stopped working later. But maybe some of the back and forth it does with the thing caused it to fault after a while because it was set to RS-8.

**Comment by benjackson1111** [comment]
seems to be good now. thanks.

**Comment by sfeakes** [comment]
Glad it's working and you sorted it out.
